ocs_ci/helpers/dr_helpers.py

```
@@ -4453,3 +4484,265 @@ def create_cdi_cert_configmap(namespace, configmap_name="user-ca-bundle"):
+    ocs_ci/utility/proxy.py for runtime kubeconfig modifications.
+
+    The mapping is:
+        - hub: first ACM cluster
+        - c1, c2: first and second non-ACM (managed) clusters
+        - passive-hub: recovery cluster (if configured)
+
+    """
+    from ocs_ci.ocs.utils import get_all_acm_indexes, get_non_acm_cluster_config
+
+    config_path = constants.ODF_CLI_DR_CONFIG_PATH
+    with open(config_path, "r") as f:
```

Ruff (0.16.0)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)